strapi/strapi
# feat: add history document action

### What does it do?

- removes the History button that was in an injection zone
- implement it using a header document action instead
- show "new" tag only for this document action

### Why is it needed?

it's the designs

### How to test it?

- open your own `api::` content types in the CM, find the button in the top right overflow menu, click on content history, you should be redirected to the history for that entry
- open an entry from a content type created by a plugin, the content history menu item should be disabled
- the menu item should be disabled when you're creating an entry (including when adding a new locale to an already saved document)
- disable content history via the future flag, you should not see the document action at all
